interchange/DeviceResources.capnp

```diff
@@ -249,6 +271,7 @@ struct Device {
     cell          @0 : StringIdx $stringRef();
     commonPins    @1 : List(CommonCellBelPinMaps);
     parameterPins @2 : List(ParameterCellBelPinMaps);
+    pinsDelay     @3 : List(PinsDelay);
```

Comment:
I think `pinsDelay` should be a field of `CommonCellBelPinMaps` and `ParameterCellBelPinMaps`. This is because timing can vary both based on placement (for example, in a Xilinx device between the 5LUT and 6LUT), and parameterisation.

Although we do have an occasional need for timing information before placement as discussed earlier, I think picking the maximum across all placements and using this will work so timing information without placement isn't something that needs to be explicitly specified.

Reply:
We could attach R per distance and C per distance to each wire. I would express the distance in multiples of the tile size (1.0 means one tile apart). This way we won't loose much information and depending on the complexity of the final timing model we want we can collapse this data to per-node RC delays.

Reply:
I think it's probably worth doing some experiments on the xc7 data (which uses per-tile-wire data, right?) to compare the relative error of the different methods.